### PR TITLE
[EDMTP-417] Added typeHint and deleteHint to mutlifield

### PR DESCRIPTION
**Pull request by Alexey-Chernyakov**

### CODE REVIEW

**Comment by Alexey-Chernyakov** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Alexey-Chernyakov** [comment]
recheck

**Comment by Konyaka1** [comment]
I have read the CLA Document and I hereby sign the CLA

File: core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/MultiField.java [hunk 1]
```diff
@@ -46,4 +49,21 @@
      * @return Reference to a class
      */
     Class<?> value() default _Default.class;
+
+    /**
+     * Maps to the {@code deleteHint} attribute of this Granite UI component's node.
+     * If set to true, generate the <a href="https://sling.apache.org/documentation/bundles/manipulating-content-the-slingpostservlet-servlets-post.html#delete">
```
Comment (smiakchilo) [inline core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/MultiField.java:55]: (cosmetic) generate**s**

File: core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/MultiField.java [hunk 2]
```diff
@@ -46,4 +49,21 @@
...
+     * {@code @Delete}</a> attribute for the http request
+     * @return True or false
+     */
+    @PropertyRendering(ignoreValues = "false")
+    boolean deleteHint() default false;
+
+    /**
+     * Maps to the {@code typeHint} attribute of this Granite UI component's node.
+     * Used to specify value of <a href="https://sling.apache.org/documentation/bundles/manipulating-content-the-slingpostservlet-servlets-post.html#typehint">
```
Comment (smiakchilo) [inline core/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/MultiField.java:64]: (cosmetic) **the** value, and then, wrap `@TypeHint` in `{@code }` the way you did for @Delete above

**Review by smiakchilo — APPROVED** [review]

**Pull request merged**